packaging/linux/flatpak/dev.lizardbyte.sunshine.yml

```diff
@@ -41,69 +41,6 @@ modules:
         url: http://archive.ubuntu.com/ubuntu/pool/main/b/boost1.74/boost1.74_1.74.0.orig.tar.xz
         sha256: 2467be4af625b5ae4b3c93fc7af196a09eba39c11a7338cd9e8b356fa44d2f45
 
-  - name: ffmpeg
```

Comment:
I made assumptions that all of these additional ffmpeg-related build steps should be obsolete but I don't know much about the different packaging systems

Reply:
For now might need to keep this one in flatpak, but only for the aarch64 build. I'm not an expert on the flatpak build, but I think there's a way to specify it for only a certain architecture. @istori1 can probably shed some insight.